tools/ocean/copernicusmarine.xml

```diff
@@ -0,0 +1,94 @@
+            <sanitizer invalid_char="">
+                <valid initial="string.letters,string.digits">
+                    <add value="_" />
+                    <add value="-" />
+                    <add value=" " />
+                    <add value=":" />
+                    <add value="." />
+                </valid>
+            </sanitizer>
+            <validator type="regex">[0-9a-zA-Z_]+</validator>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="output_netcdf" label="Copernicus marine data" from_work_dir="./*.nc" format="netcdf"/>
+    </outputs>
+    <tests>
+        <test expect_failure="true" expect_exit_code="1">
+            <param name="input_text" value="copernicusmarine subset --dataset-id cmems_mod_med_phy_my_4.2km-climatology_P1M-m --variable bottomT_avg --variable bottomT_std --variable mlotst_avg --variable mlotst_std --variable so_avg --variable so_std --variable thetao_avg --variable thetao_std --variable uo_avg --variable uo_std --variable vo_avg --variable vo_std --variable zos_avg --variable zos_std --start-datetime 1993-12-01T00:00:00 --end-datetime 1993-12-01T00:00:00 --minimum-longitude 10.283266521135577 --maximum-longitude 12.139348881644054 --minimum-latitude 38.461936025366256 --maximum-latitude 39.82957565942514 --minimum-depth 1.0182366371154785 --maximum-depth 1.0182366371154785"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+
+        **Copernicus Marine Data Store**
+=======================================================================================================
+
+** Context **
+This tool is a wrapper to retrieve data from the Copernicus Marine Environment Monitoring Service (CMEMS).
+
+- It allows to retrieve data from the Copernicus Marine Service.
+- Any user willing to use this tool needs to `create a new account <https://data.marine.copernicus.eu/login>`_.
+- Set your Copernicus CMEMS API Key via: User > Preferences > Manage Information
+- Enter your username and password for Copernicus CMEMS
+- Compose your request directly on Copernicus Marine Data Store 
+    - Choose there which data interest you click on the download button
+    - Then on the top right click again on the big download butto
+    - Log in
+    - Click on "Automate"
+    - You should have a pop-up window called "Automate download"
+    - Copy the ">_Command-Line Interface" proposed there
+- Back on Galaxy paste it in the input field "Paste API Request".
+
+For more information on the Command-Line Interface (CLI) go on `Copernicus Marine Toolbox CLI - Subset <https://help.marine.copernicus.eu/en/articles/7972861-copernicus-marine-toolbox-cli-subset>`
+
+** Input **
+Command line from the Copernicus marine services copy paste as a text.
+
+** Output **
+A netcdf file containing the the data chose by the user from the Copernicus Marine Data Store
+
+
+    ]]></help>
+    <citations>
+        <citation type="bibtext"> 
```

Comment:
```suggestion
        <citation type="bibtex"> 
```

The linting errors are valid ;)

Reply:
ahah no I was refering to the python script linting ;) ( Did not see this one)